Firestore/Source/API/FIRDocumentChange.mm

```diff
@@ -38,12 +40,12 @@ - (instancetype)initWithType:(FIRDocumentChangeType)type
 @implementation FIRDocumentChange (Internal)
 
 + (FIRDocumentChangeType)documentChangeTypeForChange:(FSTDocumentViewChange *)change {
-  if (change.type == FSTDocumentViewChangeTypeAdded) {
+  if (change.type == DocumentViewChangeType::Added) {
```

Comment:
Since this is now an enum (well; I guess it was before too), we could switch this to an exhaustive switch statement. (That way, the compiler will notice when we forget to handle a new case.)

But, although trivial, this may be more involved than you'd like for an otherwise trivial-er PR.

Options:
1) switch to switch.
2) switch to switch in a followup.
3) don't worry about it. It may not be worth the effort.

Any of the above are fine by me. If you choose 1 or 2, obviously do so throughout rather than just here.

Reply:
Re throughout: Yeah, you're right; I think this is the only instance where this fits in this PR. (I skimmed some other if statements involving these enums without reading them close enough; they don't quite fit.)